backend/app/api/endpoints/projects.py

```
@@ -140,6 +142,59 @@ def archive_all_project_chats_endpoint(
         )
 
 
+@router.get("/worktrees", response_model=ProjectWorktreeListResponse)
+async def list_project_worktrees_endpoint(
+    client_origin: ClientOriginQuery = CLIENT_ORIGIN_FRONTEND,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)